pkg/messaging/pubsub.go

```diff
@@ -26,6 +29,9 @@ type Subscriber interface {
 	// Unsubscribe unsubscribes from the message stream and
 	// stops consuming messages.
 	Unsubscribe(id, topic string) error
+
+	// Close gracefully closes message subscriber's connection.
+	Close()
```

Comment:
Close should return an error. In case the underlying implementation is not returning it, return nil.